.travis.yml

```diff
@@ -1,15 +1,12 @@
-  - npm install -g bower polylint web-component-tester
-  - bower install
-  - polylint
+  - npm install -g polymer-cli
+  - polymer install --variants
 env:
   global:
-    - secure: >-
-        Qe0WSR8xqRBZgXFr5yzHQBlcG2g4y4N0tH0rDCbFU4/Nf75ADDf5/8bWGbI9hAIyXj8CP27BkZvPClx6vLm+m45yjNBRXuUw5YrSSjqKMNxIGx7gajXYbQn/LRgv7e9t7H0htYXWZ2/1wEDtoqlrTL8oKGvGfpRHFkeV00kYPKqG7BgZBaXNc9HJIXHKW7Pq8kqPR9zlqvmIskrStZ7VapldfgfMWOuI1Tl5i65+YuGT2IDsvaykVhwjP6dtTrJeE4ACt8jN+l7vRoyoEXZDfCZQoBXPhomMxr0g1wrSHohG3PW+IMcyUy0sMgYLBmGuxENsdSIzlzlyVpWt2U1+5RQY5cEh7P7616pCZx8VLjhsTh4plLrx87ycTEvjTw91OJJzlSQYP55RmZJWQs9rRHyKHZf6n4WRyiG+7+8DfSCs8lAvpohHFvG2y28sxSWctKVmGjK0h6k/Q/sCxCxRN4+Bq5ByHCeuE0VsVyzSVPJYw49VE1N3vQQxYioEuxxI1y6y8EORBOK7hf1QoVKRPtmJ4keWybkLAhNRFhAlaSmaRyI2A+q5yyLSdk8LrxP93GJxQe0bI0sU3V65jrDRHWtdjDK+2tzhwLWkbxt0eZiywz77b/q0BY6eKddJ40zZTdgXIZ4Es4gxM9mZYRAmTcKVGoYMNPJCD3ep3213w1Y=
-    - secure: >-
-        OJAW+ssXIXb+2Ze4skRm4QrFY6OcpFgKHRhcdT5TGv6OpzHPZO5wWgX37Ocb2FtgGbiIAMiI1B6dR4C998LGOUVO7kGA1f7vnp75PvsvYON1sKLfAyuIBJuI6uLRE6jOSUDAG9Rc3Q3Hv33QOnDcC1uwuYDFIF8pjn++TPqx3c0WwapD+IBsnHhFsTi3H7d0vwJNRUprAZq4leo+3f/jz4PMWlV7IxrjG0ghseOO4sEXDLkm6GXeaJV78HCoNFk/14LC947BxN6T5Sc9lzWo/s0t8JoZODjAe6uON9OzfgsCs8oH7hhNys8QM/6EqBgcL2sEvJi9N0vsBKG351DHVFjGMv7j28+N/FzYkrrtET6EyPxcZ7S3TYUQb92hW/cMBJ6lqusmGfzFQ56WnAJyKJei0vH+IykBFBJ+Km2X3DS3/i30l0j8w4dN0vjYkAC+oWGho5iDN8cp5K93cLhgnDhPr6fU2tRMHB3bPjRMGgT+TmP9zJJ87mSiwCJgJJGl8NVODz54e66CZcOPZdTBzsdC7YSfvlvUE7PUt77LJO69EfZT3s3CAfe0hks7fFpZ8SIqw50F2K5ed7o4CswewPlvsA0yeZCOHZWFk0X2rRSWzfgz7TOAfim37HYp/kYQWFi8XYpd93Q5uPSVId6CX/dJoGfFNHx4oY/lsKQKbYo=
+    - secure: Qe0WSR8xqRBZgXFr5yzHQBlcG2g4y4N0tH0rDCbFU4/Nf75ADDf5/8bWGbI9hAIyXj8CP27BkZvPClx6vLm+m45yjNBRXuUw5YrSSjqKMNxIGx7gajXYbQn/LRgv7e9t7H0htYXWZ2/1wEDtoqlrTL8oKGvGfpRHFkeV00kYPKqG7BgZBaXNc9HJIXHKW7Pq8kqPR9zlqvmIskrStZ7VapldfgfMWOuI1Tl5i65+YuGT2IDsvaykVhwjP6dtTrJeE4ACt8jN+l7vRoyoEXZDfCZQoBXPhomMxr0g1wrSHohG3PW+IMcyUy0sMgYLBmGuxENsdSIzlzlyVpWt2U1+5RQY5cEh7P7616pCZx8VLjhsTh4plLrx87ycTEvjTw91OJJzlSQYP55RmZJWQs9rRHyKHZf6n4WRyiG+7+8DfSCs8lAvpohHFvG2y28sxSWctKVmGjK0h6k/Q/sCxCxRN4+Bq5ByHCeuE0VsVyzSVPJYw49VE1N3vQQxYioEuxxI1y6y8EORBOK7hf1QoVKRPtmJ4keWybkLAhNRFhAlaSmaRyI2A+q5yyLSdk8LrxP93GJxQe0bI0sU3V65jrDRHWtdjDK+2tzhwLWkbxt0eZiywz77b/q0BY6eKddJ40zZTdgXIZ4Es4gxM9mZYRAmTcKVGoYMNPJCD3ep3213w1Y=
+    - secure: OJAW+ssXIXb+2Ze4skRm4QrFY6OcpFgKHRhcdT5TGv6OpzHPZO5wWgX37Ocb2FtgGbiIAMiI1B6dR4C998LGOUVO7kGA1f7vnp75PvsvYON1sKLfAyuIBJuI6uLRE6jOSUDAG9Rc3Q3Hv33QOnDcC1uwuYDFIF8pjn++TPqx3c0WwapD+IBsnHhFsTi3H7d0vwJNRUprAZq4leo+3f/jz4PMWlV7IxrjG0ghseOO4sEXDLkm6GXeaJV78HCoNFk/14LC947BxN6T5Sc9lzWo/s0t8JoZODjAe6uON9OzfgsCs8oH7hhNys8QM/6EqBgcL2sEvJi9N0vsBKG351DHVFjGMv7j28+N/FzYkrrtET6EyPxcZ7S3TYUQb92hW/cMBJ6lqusmGfzFQ56WnAJyKJei0vH+IykBFBJ+Km2X3DS3/i30l0j8w4dN0vjYkAC+oWGho5iDN8cp5K93cLhgnDhPr6fU2tRMHB3bPjRMGgT+TmP9zJJ87mSiwCJgJJGl8NVODz54e66CZcOPZdTBzsdC7YSfvlvUE7PUt77LJO69EfZT3s3CAfe0hks7fFpZ8SIqw50F2K5ed7o4CswewPlvsA0yeZCOHZWFk0X2rRSWzfgz7TOAfim37HYp/kYQWFi8XYpd93Q5uPSVId6CX/dJoGfFNHx4oY/lsKQKbYo=
```

Comment:
these were supposed to change in #60, not sure if they still need to change or not